src/libraries/black-listed-key-bindings.library.ts

```diff
@@ -6,23 +6,23 @@ import {
 } from './listener.library';
 
 export const blacklistedKeyCombinations = [
```

Comment:
did you test this change?  I am asking because I think the matching assumes all keys have been changed to lower case.  Please verify this still works after changing these to upper case.

Reply:
Fixed this, forgot to change back to lowercase, was testing for visual purposes